packages/javascript-sdk/src/session-manager/index.ts

```diff
@@ -28,8 +28,8 @@ abstract class SessionManager {
     const init: RequestInit = {
       credentials: 'include',
       headers: new Headers({
-        'accept-api-version': 'protocol=1.0,resource=2.0',
-        'x-requested-with': REQUESTED_WITH,
+        'Accept-Api-Version': 'protocol=1.0,resource=2.0',
```

Comment:
Is "Accept-Api-Version" the correct casing for this header? In the ticket Brandon mentions "Accept-API-Version", so I just wanted to get confirmation which one is correct

Reply:
I can make the change to reflect the request in the ticket.